packages/scene-composer/src/components/three-fiber/EditorCamera.tsx

```diff
@@ -152,6 +153,8 @@ export const EditorMainCamera = forwardRef<Camera>((_, forwardedRef) => {
       let position = DEFAULT_CAMERA_POSITION;
       let target = DEFAULT_CAMERA_TARGET;
 
+      console.log('camera command!');
```

Comment:
Same question here. Why are we adding logs? If it's intentional, the messaging should go through localization so it can be meaningful to all users.

Reply:
removed